src/backend/gpopt/gpdbwrappers.cpp

```diff
@@ -2721,4 +2723,34 @@ gpdb::GPDBAllocSetContextCreate()
 	return NULL;
 }
 
+TransactionId
+gpdb::GetTupleHeaderXmin(HeapTupleHeaderData *header)
+{
+	GP_WRAP_START;
```

Comment:
I think these wrappers are redundant, because... no one generates errors inside them.

Reply:
Wrappers are only needed if the errors are generated? There are a lot of similar functions inside the gpdbwrappers.cpp, why we shouldn't do everything in a uniform manner?

Reply:
> There are a lot of similar functions inside the gpdbwrappers.cpp

Yes, it looks like they went overboard with catching non-existent errors =)

Reply:
> Have you seen postgres functions used by ORCA without a wrapper?

No, that’s why I wrote above that I didn’t see C functions being directly called from pluses.

Reply:
I share your point of view, I think it's unnessecary too.  I'm mostly concerned about non-uniform way of C functions handling, about unconsoledated aproach.